src/jvm/test/resources/expected_logs/custom_threads_trace.txt

```diff
@@ -0,0 +1,84 @@
+|                                                                                                    |                                                                                                                       |                                                                                                                       |   value.READ: 11 at CustomThreadsRepresentationTest.basic$lambda$1(CustomThreadsRepresentationTest.kt:43)             |
+|                                                                                                    |                                                                                                                       |                                                                                                                       |   value.WRITE(12) at CustomThreadsRepresentationTest.basic$lambda$1(CustomThreadsRepresentationTest.kt:43)            |
+|                                                                                                    |                                                                                                                       |                                                                                                                       |   MONITOREXIT at CustomThreadsRepresentationTest.basic$lambda$1(CustomThreadsRepresentationTest.kt:42)                |
+|                                                                                                    |                                                                                                                       |                                                                                                                       |   result: void                                                                                                        |
+|   Thread#1.join() at CustomThreadsRepresentationTest.basic(CustomThreadsRepresentationTest.kt:48)  |                                                                                                                       |                                                                                                                       |                                                                                                                       |
+|     join Thread 2                                                                                  |                                                                                                                       |                                                                                                                       |                                                                                                                       |
```

Comment:
Do you mean `start Thread N` and `join Thread N` lines?

They correspond to `ThreadStartTracePoint` and `ThreadJoinTracePoint` objects.
If I don't add these objects to the trace, then no stack frames related to start/join are going to be printed at all (e.g. `Thread#2.start()` or `Thread#2.join()` are going to be printed). 
This is because currently, method calls are only added in the trace if there are some trace points inside.

Reply:
Is it possible to process these method calls similarly to the atomic ones?

Reply:
I can try this approach.

Reply:
The approach with atomic methods did not work out. Because if we treat start/join as atomic methods, the managed strategy also starts to insert switch points before them (and previously we agreed to get rid of exactly this kind of redundant switch points before start/join). 

However, I've implemented alternative approach relatively easy. 
I still add `ThreadStartTracePoint` and `ThreadJoinTracePoint` to the trace, but then I'm simply do not print them.